go-controller/pkg/ovn/pods.go

```diff
@@ -184,7 +186,14 @@ func getRoutesGatewayIP(pod *kapi.Pod, gatewayIPnet *net.IPNet) ([]util.PodRoute
 			routes = append(routes, route)
 		}
 	} else {
-		gatewayIP = gatewayIPnet.IP
+		if namespaceExternalGW == "" {
+			gatewayIP = gatewayIPnet.IP
+		} else {
+			// We use .3 as gateway as the namespace contains namespaceExternalGW annotation for. The value of the annotation
+			// will be used in the hybrid overlay code for setting the outer destination on VXLAN packets
+			gatewayCIDR, _ := util.GetNodeWellKnownAddresses(gatewayIPnet)
```

Comment:
@rcarrillocruz don't we want the .3 address here?

Reply:
Also let's get https://github.com/ovn-org/ovn-kubernetes/pull/1250 merged and then you can call the right function.

Reply:
oops , that's right. Alright, will review and keep an eye on your PR